inferentia/README.md

```diff
@@ -96,9 +96,18 @@ Once the TorchScript model supporting Inferentia is obtained, use the [gen_trito
 An example invocation for the `gen_triton_model.py` can look like:
 
 ```
- $python3 inferentia/scripts/gen_triton_model.py --triton_input INPUT0,FP16,4x384 INPUT1,FP16,4x384 INPUT2,FP16,4x384 --triton_output OUTPUT0,FP16,4x384 OUTPUT1,FP16,4x384 --compiled_model /home/ubuntu/bert_large_mlperf_neuron_hack_bs1_dynamic.pt --triton_model_dir bert-large-mlperf-bs1x4
+ $python3 inferentia/scripts/gen_triton_model.py --triton_input INPUT__0,INT64,4x384 INPUT__1,INT64,4x384 INPUT__2,INT64,4x384 --triton_output OUTPUT__0,INT64,4x384 OUTPUT__1,INT64,4x384 --compiled_model /home/ubuntu/bert_large_mlperf_neuron_hack_bs1_dynamic.pt --neuron_core_range 0:3 --triton_model_dir bert-large-mlperf-bs1x4
 ```
 
+NOTE: As the compiled torch script model does not carry sufficient
+model metadata, user is expected to follow naming convention of

```

Comment:
the user must ensure the inputs and outputs in the model configuration follow a specific naming convention i.e. `<name>__<index>`. Use the examples from https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#inputs-and-outputs to make it clear.